### ISSUE TITLE
SNMP RFC error when building RPM 

### ISSUE THREAD

**Issue by titansmc** [acassen/keepalived#891]
Hi,
I am trying to build RPM's packages for keepalived 2.0.0 but I get this error... Am I missing any library ? I've just installed everything that was required.

When compiling and installing the software didn't fail, but building packages with the provided spec file it failed.

```
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd keepalived-2.0.0
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ exit 0
Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.uXFq7W
+ umask 022
+ cd /root/rpmbuild/BUILD
+ cd keepalived-2.0.0
+ CONFIG_OPTS=
+ '@SNMP_RFC_TRUE@CONFIG_OPTS= --enable-snmp-rfc'
/var/tmp/rpm-tmp.uXFq7W: line 35: @SNMP_RFC_TRUE@CONFIG_OPTS= --enable-snmp-rfc: command not found
error: Bad exit status from /var/tmp/rpm-tmp.uXFq7W (%build)


RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.uXFq7W (%build)
```

**Comment by fenice2** [comment]
I've also hit the same problem, I don't know if this is the correct answer (I'm sure it's not :)) but I commented out this line:

@SNMP_RFC_TRUE@CONFIG_OPTS= --enable-snmp-rfc

in the spec file. Keepalived  builds correctly and it's installed and started on my CentOS7 server. I'll try and get some time tomorrow to see if it actually works correctly.

**Comment by pqarmitage** [comment]
The above edit is not the way to go, since keepalived.spec is generated from keepalived.spec.in.

It transpired that SNMP_RFC was not being defined in configure.ac. Commit ca23497 should resolve the problem.

**Comment by fenice2** [comment]
Yes, I realised that wasn't the correct 'fix' but not being a coder my first reaction is always to comment out a troublesome statement. Thanks for the quick fix. :)
